src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/QTClusterFinder.h

```diff
@@ -147,12 +162,41 @@ namespace OpenMS
     void setParameters_(double max_intensity, double max_mz);
 
     /// Generates a consensus feature from the best cluster and updates the clustering
-    void makeConsensusFeature_(std::list<QTCluster>& clustering,
+    bool makeConsensusFeature_(Heap& cluster_heads,
```

Comment:
I know this has also not been done before, but could you document the private functions members?
For some, its even hard to see what is input, and what is output parameter. Can some be made const to make this more clear? Sometimes, ordering them like, input1, input2, output1, output2 also makes the interface a bit easier to read.
This goes for all functions here...

Reply:
All the new methods we made for this class are now well documented with their parameters.